web-common/src/components/column-profile/ColumnProfile.svelte

```diff
@@ -113,6 +113,7 @@
         {hideRight}
         {hideNullPercentage}
         {compact}
+        enableProfiling={!$profileColumns?.isFetching}
```

Comment:
Would it work with `$profileColumns?.isSuccess` or `$profileColumns?.data`? Either of those would be more semantically clear, IMO.

Reply:
`enableProfiling` has to be false when we are refetching `profileColumns`. `isSuccess` is true even while the `profileColumns` is being refetched.